modules/ROOT/pages/runtime-monitoring.adoc

```diff
@@ -1,11 +1,16 @@
 = Bonita Runtime Monitoring
 :description: Discover how to monitor a runtime environment running Bonita
 
+[NOTE]
+====
+
```

Comment:
For the record, the blank line isn't necessary. Many blank lines have been added during the conversion from mb, but it's useless.